pkg/loader/compose/compose.go

```diff
@@ -120,6 +121,28 @@ func checkUnsupportedKey(composeProject *project.Project) []string {
 							yamlTagName = "networks"
 						}
 					}
+
+					if f.Name() == "Links" {
+						//Links has "SERVICE:ALIAS" style, we don't support SERVICE != ALIAS
+						if linksArray := val.FieldByName(f.Name()); linksArray.Kind() == reflect.Slice {
```

Comment:
nested if, may be better to do `if f.Name() == "Links" && if linksArray := val.FieldByName(f.Name()); linksArray.Kind() == reflect.Slice {`

Reply:
Done